src/components/header.vue

```diff
@@ -0,0 +1,32 @@
+<template>
+    <div>
+        <nav class="navbar navbar-expand-lg navbar-light bg-light">
+            <router-link to="/" >To-Do-App</router-link>
```

Comment:
Trebalo bi da sve rute imaju ime, jer su URL-ovi skloni promenama.
Onda bi se kretao ovako `:to={ name: 'home' }`

Reply:
Ovo vazi za svaki `<router-link>`